content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -20,11 +20,15 @@ This page provides an overview of authenticating.
 All Kubernetes clusters have two categories of users: service accounts managed
 by Kubernetes, and normal users.
 
-Normal users are assumed to be managed by an outside, independent service. An
-admin distributing private keys, a user store like Keystone or Google Accounts,
-even a file with a list of usernames and passwords. In this regard, _Kubernetes
-does not have objects which represent normal user accounts._ Normal users
-cannot be added to a cluster through an API call.
+Normal users are assumed to be managed by
+
+- an outside, independent service
```

Comment:
I think this should be:
```suggestion
Normal users are assumed to be managed by an outside, independent service, such as:
```

Reply:
@tengqm have you had the opportunity to review this comment? It makes sense to me and seems like this PR would be ready to merge after.

Reply:
@jimangel @sftim, I did reviewed the feedback and I don't think the enumerated items are all "services".